public/components/CampaignAnalytics/Analytics/CampaignReferral.js

```diff
@@ -5,18 +5,21 @@ import "react-infinity-menu/src/infinity-menu.css";
 export default class CampaignReferral extends React.Component {
 
   render() {
+    // The keyPath needs to be halved because it is typically something like:
+    // '0.children.0.children.0.children.0.children.0'
+    // should give a depth of 4.
     const depth = Math.floor(this.props.data.keyPath.split(".").length / 2);
```

Comment:
Depth is actually available as an arg in the nodeClick method, but I've no idea how to pass it through.